### PR TITLE
chore: improve SE and fix commit hooks

### PR DESCRIPTION
**Pull request by ImRodry**
**Describe the changes in this pull request:**

This PR *actually* improves SE and fixes the commit hook
Closes #5645

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,11 @@
 	"license": "MPL-2.0",
 	"scripts": {
 		"lint": "prettier --write . && eslint --fix .",
-		"lint:ci": "eslint .",
+		"lint:ci": "prettier --check . && eslint .",
 		"inc": "tsc -p ./util/configs/tsconfig.massVer.json && node ./util/tools/massVer.js",
-		"se": "tsc -p ./util/configs/tsconfig.auto.syntaxEnforcer.json && node ./util/tools/auto/syntaxEnforcer.js",
+		"incChanged": "tsc -p ./util/configs/tsconfig.auto.syntaxEnforcer.json && node ./util/tools/bumpChanged",
```
Comment (Bas950) [inline package.json:10]:
```suggestion
		"bumpChanged": "tsc -p ./util/configs/tsconfig.auto.syntaxEnforcer.json && node ./util/tools/bumpChanged",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
change it to match file name

File: package.json (hunk 1 above)
Comment (ImRodry) [reply package.json:10]: No bc consistency

**Comment by EncryptedDev** [comment]
You changed all those metadata's but never bumped the version?

File: package.json (hunk 1 above)
Comment (Bas950) [reply package.json]: I would either change the file name or the package script name. So they are the same like all other scripts.

Comment (ImRodry) [reply package.json]: For the same reason that massVer is called inc in the script, this is called incChanged as it does the same thing to the files in the diff.

Comment (Bas950) [reply package.json]: Well while you are at it you can make the scripts and file names consistent

Comment (ImRodry) [reply package.json]: That’s unnecessary, inc and incChanged are much easier to understand that two letter acronyms

**Comment by ImRodry** [comment]
Realized it's not possible to use prettier/eslint on json files since bumping the versions would undo the changes with JSON.stringify. I added *.json to the prettier and eslint ignore files and only formatted 1 file. This is ready to be merged now.

**Comment by Bas950** [comment]
> Realized it's not possible to use prettier/eslint on json files since bumping the versions would undo the changes with JSON.stringify. I added *.json to the prettier and eslint ignore files and only formatted 1 file. This is ready to be merged now.

cant you after bumping only run it on json files?

**Comment by ImRodry** [comment]
You’d need to be running it twice, there’s no point and this format is just fine

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**